episodes/chap1_introduction_FL.md

```diff
@@ -101,6 +101,9 @@ In this course we will focus on federated analysis. In a federated setting, the
  The system sends a query or instruction to the data and only the results will get back to the user.
 The results are often akin to a form of _aggregation_ of the data. This can be in the shape of
 traditional statistics like the mean, or it can be more intricate like a machine learning model.
+When data is aggregated, it doesn't have a direct link with an individual anymore. Since the
+aggregated data cannot be be traced back to the individuals from the dataset, the privacy is 
```

Comment:
I would add the word _directly_ here. As you can do clever things to figure out individual data from aggregates.